pkg/didcomm/protocol/didexchange/service.go

```diff
@@ -238,7 +228,7 @@ func (s *Service) nextState(msgType, thID string) (state, error) {
 	return next, nil
 }
 
-func (s *Service) handle(msg *message) error {
+func (s *Service) handle(msg *message, aEvent chan<- service.DIDCommAction) error { //nolint funlen
```

Comment:
maybe the function should be split ;)

Reply:
When I look at this [function](https://github.com/hyperledger/aries-framework-go/blob/c3fbb046885a794db280ed57a43d363cda1b2742/pkg/didcomm/protocol/didexchange/service.go#L231), there are 20 lines for struct field value setting, 7 lines for return error, 5 lines for logging. There are no logic code(it's invoking other functions). So not sure it would make much sense to split this function.

Reply:
Is the suggestion to have a longer funlen maximum ... or this one just happens to be an exception?

Reply:
more of an exception. But yeah, we might need to increase the funclen config as we have wsl linter now.